uaclient/timer/__init__.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from subprocess import TimeoutExpired
+
+from uaclient import exceptions, system
+
+LOG = logging.getLogger("uaclient.timer")
```

Comment:
Portability nit:

```suggestion
LOG = logging.getLogger(__package__)
```

Reply:
Good point! Is there a reason to prefer `__package__` over `__name__`?